specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/examples/VirtualHubRouteTableV2Put.json

```diff
@@ -6,9 +6,6 @@
     "subscriptionId": "subid",
     "virtualHubName": "virtualHub1",
     "virtualHubRouteTableV2Parameters": {
-      "tags": {
```

Comment:
Are you sure this resource doesn't support tags? I think all tracked resources do that even if they are not top-level

Reply:
Adding the tags to the specs did not work either, so going to remove it from the example.

Reply:
Since this is a child resource, the tags for it will be propagated from it's parent resource